timers.js

```diff
@@ -1,4 +1,35 @@
+var seconds = 0;
+var timerId = 0;
+var timerDisplay = document.getElementById("timer");
+var numClicks = 0;
```

Comment:
It looks like you're using `numClicks` to determine whether the timer is already running. A boolean would make more sense here (e.g., `var isRunning`). You can toggle between true/false everytime you start/stop/reset the timer. Your conditionals will also look simpler (e.g., `if( isRunning ){ ... }`).